oeps/oep-1.rst

```diff
@@ -0,0 +1,266 @@
+for changing Open edX that have been rejected for various reasons. Asking
+the Open edX community first if an idea is original helps prevent too much
+time being spent on something that is guaranteed to be rejected based on
+prior discussions (searching the internet does not always do the trick).
+It also helps to make sure the idea is applicable to the entire community
+and not just the author. Just because an idea sounds good to the author
+does not mean it will work for most Open edX installations.
+
+Once the champion has asked the Open edX community as to whether an idea
+has any chance of acceptance, a draft OEP should be submitted as a pull request
+against the `central OEP repository`_.
+
+.. _central OEP repository: http://github.com/edx/open-edx-proposals
+
+As updates are necessary, the OEP author can update the pull request.
+
+OEPs can be paired with pull requests to other Open edX repositiories with
+reference implementations.
+
+OEP Review & Resolution
+-----------------------
+
+Once the authors have completed an OEP, they may request a review from
+OEP reviewers. These reviewers will select one member to act as the final
+arbiter on the OEP. This arbiter will be recorded in the "Arbiter"
+header on the OEP. The remaining reviewers and the rest of the Open edX
+community will be given the opportunity to comment on the OEP, with the
+Arbiter serving to keep the discussion on track and to evaluate when
+it has reached a final conclusion.
+
+For an OEP to be accepted it must meet certain minimum criteria. It must be a
+clear and complete description of the proposed enhancement. The enhancement
+must represent a net improvement. The proposed implementation, if applicable,
+must be solid and must not complicate Open edX unduly.
+
+An OEP can be assigned the status "Deferred". The OEP author or Arbiter can
+assign the OEP this status when no progress is being made on the OEP. Once a
+OEP is deferred, an OEP editor can re-assign it to draft status.
+
+An OEP can also be "Rejected". Perhaps after all is said and done it was not a
+good idea. It is still important to have a record of this fact. The "Withdrawn"
+status is similar - it means that the OEP author themselves has decided that
+the OEP is actually a bad idea, or has accepted that a competing proposal is a
+better alternative.
+
+When a OEP is Accepted, Rejected or Withdrawn, the OEP should be updated
+accordingly. In addition to updating the status field, at the very least the
+Resolution header should be added with a link to the relevant post in the edx-code
+mailing list archive or to the appropriate section of the PR.
+
+OEPs can also be superseded by a different OEP, rendering the original
+obsolete.
+
+The possible paths of the status of OEPs are as follows:
+
+.. image:: oep-1/state-flow.png
+
+OEP Maintenance
+---------------
+
+In general, OEPs are no longer modified after they have reached the Final state.
+They can be replaced by subsequent OEPs, however.
```

Comment:
Ah, nevermind I see you already have a status of "Replaced", it may be worth calling that out explicitly here.